services/src/api/handlers/layers.rs

```diff
@@ -40,6 +41,25 @@ where
         web::scope("/layers")
             .service(
                 web::scope("/collections")
+                    .service(
+                        web::scope("/search")
+                            .route(
+                                r#"autocomplete/{provider}/{collection}"#,
+                                web::get().to(autocomplete_handler::<C>),
+                            )
+                            .route(
+                                r#"capabilities/{provider}"#,
```

Comment:
wouldn't it make more sense to have this under /layers/{provider}/capabilities?

Reply:
:+1: